lib/hpke/src/hpke.cpp

```diff
@@ -235,6 +228,14 @@ AEAD::get<AEAD::ID::CHACHA20_POLY1305>()
   return AEADCipher::get<AEAD::ID::CHACHA20_POLY1305>();
 }
 
+template<>
+const AEAD&
+AEAD::get<AEAD::ID::export_only>()
+{
+  return ExportOnlyCipher::get();
```

Comment:
What about creating static object here instead of making ExportOnlyCipher as singleton ?

Reply:
Do you mean you would prefer if the singleton were a static member of `ExportOnlyCipher`, like with `::instance` on other types here?

Reply:
I mean something like this :

> template<>
> const AEAD&
> AEAD::get<AEAD::ID::export_only>()
> {
>     static ExportOnlyCipher eoc;
>     return eoc;
> }

and make ExportOnlyCipher not a singleton class.